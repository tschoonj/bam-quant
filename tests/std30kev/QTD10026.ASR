File: C:\xrfdaten\2014KW30\chris\std30kev\std10026.spe
Start: 22.07.2014 10:01:18
Stop: 22.07.2014 10:02:59
BESSY: 298.153382 298.9709553
IONI: 2.003979E-11 2.006964E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 98036
Run: 100000
Dead: 2
ICR: 170
OCR: 0
Accumulated BESSY: 59654.1844243
Accumulated Ioni: 4.00950098E-09
N Accumulation: 200
Normfactor:  5.0880818e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Fe
exp_x 152.720397949219 152.7204
exp_y 150.465789794922 150.4658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  26   1      6.399         3.         5.        .76
