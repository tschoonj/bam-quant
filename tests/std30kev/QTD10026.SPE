$DATA:
0 2000
23051
14710
6563
1898
364
48
6
0
0
0
0
0
0
0
0
0
2
8
11
7
7
9
14
12
13
13
14
12
13
11
14
13
14
9
12
11
13
9
5
15
8
9
11
9
10
15
5
10
16
8
3
10
7
11
9
11
12
4
9
15
12
12
9
10
8
2
13
6
9
6
11
4
7
5
5
8
13
4
8
8
6
8
10
5
3
7
9
4
7
3
11
4
10
11
9
3
9
6
2
8
8
8
7
5
9
6
12
9
12
8
5
12
9
7
6
7
6
5
6
16
8
9
7
5
9
9
8
10
6
8
6
9
7
8
9
7
4
4
7
3
9
6
3
5
1
9
7
9
9
6
6
8
12
8
7
8
7
10
8
3
5
6
6
8
3
7
2
6
9
6
7
8
9
6
7
5
7
1
7
11
5
4
4
7
2
7
4
4
7
4
2
2
4
3
7
3
6
1
3
6
5
4
9
3
3
4
4
4
2
8
0
3
2
0
4
2
1
2
2
1
5
4
0
4
1
3
2
4
1
1
1
1
2
2
0
1
5
2
2
4
3
2
0
2
1
1
0
2
0
1
2
4
2
3
5
1
1
0
3
0
0
1
1
3
1
2
2
0
3
2
0
2
0
2
2
0
1
0
3
0
1
0
0
2
0
0
2
0
0
0
1
1
0
0
0
1
1
1
1
0
2
0
1
4
0
2
2
1
2
0
1
2
2
1
1
4
2
3
3
3
5
3
2
2
3
0
1
1
0
0
0
0
2
0
1
1
1
0
1
2
1
0
1
2
2
1
3
7
4
3
8
4
3
3
0
1
0
2
1
0
1
1
1
1
0
1
0
0
1
0
2
1
1
1
0
3
0
0
1
1
0
1
0
1
1
0
1
0
0
0
0
0
1
0
1
0
1
0
1
1
1
0
0
1
0
1
1
0
1
1
2
1
1
0
0
0
0
1
0
1
1
0
1
0
0
0
1
1
0
1
3
1
1
0
1
0
1
0
1
0
0
1
1
0
0
0
0
2
0
2
0
2
0
0
0
0
1
0
1
3
0
1
1
1
0
1
0
1
0
1
0
0
0
0
0
2
0
1
0
1
0
0
0
1
0
1
1
2
0
0
2
0
0
0
0
2
1
1
0
2
1
0
2
0
0
2
0
1
0
2
0
0
1
5
1
0
0
0
0
3
1
0
2
0
0
2
2
0
2
0
2
1
3
1
1
1
1
0
0
2
1
0
2
2
2
0
2
0
1
3
0
1
2
2
3
1
1
0
0
1
1
0
1
2
2
1
3
2
1
1
2
3
2
2
0
0
2
0
0
3
2
1
1
1
1
0
0
1
1
1
2
3
1
2
0
3
1
1
1
1
3
1
1
4
3
2
1
1
0
5
1
0
1
5
3
3
2
3
0
4
3
3
3
3
4
3
3
2
0
2
0
4
5
3
3
3
1
3
5
3
1
0
3
5
4
2
2
2
1
1
4
3
3
3
3
2
4
2
3
3
3
2
3
5
0
3
5
5
7
3
3
3
2
3
8
3
1
2
2
3
1
6
2
5
3
1
4
6
3
5
0
5
4
4
4
2
3
5
4
2
6
3
6
0
1
5
4
2
1
4
4
2
2
5
3
2
5
4
3
4
3
8
2
3
2
3
6
4
4
6
5
4
6
4
5
7
5
6
5
3
3
11
5
3
8
4
3
6
6
5
8
3
5
3
3
5
3
5
4
3
4
7
4
2
5
6
3
11
7
13
16
24
29
25
37
56
64
82
85
100
91
88
80
74
61
45
36
39
29
18
15
9
9
1
4
3
1
2
7
3
5
5
7
2
2
4
9
6
4
3
5
4
5
3
5
7
3
5
5
6
3
2
7
4
4
5
6
1
2
3
1
4
2
3
1
5
3
2
1
6
6
3
2
3
7
6
2
4
3
6
1
3
5
8
3
7
7
3
6
4
3
4
5
4
6
2
4
7
5
10
14
21
21
19
19
18
15
18
19
12
8
13
5
3
2
1
2
5
7
7
9
8
7
6
6
3
3
3
7
4
3
3
3
3
0
3
2
2
5
2
4
3
4
5
6
1
6
5
3
3
2
9
4
3
3
4
5
6
4
1
5
3
4
1
5
6
6
2
3
1
5
1
6
5
4
6
3
3
1
5
2
4
5
3
5
7
4
0
2
1
3
4
5
4
3
2
4
4
8
5
4
2
3
5
3
6
1
3
4
4
6
1
5
2
4
3
1
3
4
8
2
5
2
4
4
7
4
7
5
7
4
6
5
4
4
9
1
3
1
3
3
1
2
1
2
3
2
2
9
2
5
1
3
4
2
1
7
4
4
4
3
6
3
1
7
2
6
5
3
7
6
7
1
5
8
5
3
0
5
3
2
1
2
3
4
5
6
1
6
3
3
2
4
3
1
3
2
4
1
5
3
3
0
5
3
3
4
3
5
3
2
2
4
1
2
2
2
3
2
0
4
2
4
1
4
3
4
5
4
2
2
2
1
2
2
4
6
0
1
3
5
4
5
1
4
2
3
4
2
3
4
2
2
2
2
5
2
1
2
2
3
5
4
5
5
4
1
4
3
6
4
2
5
1
2
3
4
1
1
4
6
5
2
1
2
3
5
3
5
3
1
3
3
3
1
1
2
6
1
1
0
4
2
1
3
5
5
2
3
0
3
3
2
1
7
1
3
4
1
4
0
1
1
2
1
5
2
3
3
0
4
2
1
2
3
3
1
2
1
4
5
2
2
2
5
2
3
0
3
4
1
2
4
4
6
4
0
4
4
0
4
2
6
4
4
3
5
9
2
14
5
9
9
7
7
11
12
13
11
9
8
9
12
10
9
5
12
13
6
3
2
3
0
6
5
2
5
3
3
5
2
2
2
0
5
4
4
1
4
4
3
3
4
4
1
0
2
4
7
5
3
2
0
3
0
2
3
7
4
4
1
3
3
3
2
2
3
2
7
5
3
3
4
7
3
0
3
3
1
2
2
4
3
0
4
1
4
4
2
1
3
1
3
4
1
4
1
5
6
4
5
5
7
4
4
4
2
1
6
4
3
3
4
4
2
6
7
7
4
5
7
2
2
6
4
5
6
10
2
5
4
5
3
2
0
2
3
2
6
2
1
5
4
4
3
6
2
2
5
1
3
4
2
2
1
3
6
2
7
6
4
7
7
2
2
3
5
5
4
6
5
1
0
4
5
9
3
5
4
1
4
2
4
3
3
0
2
4
3
3
2
3
6
6
2
2
4
4
6
2
1
5
3
3
3
3
7
4
5
6
5
7
3
6
3
7
5
9
4
4
5
10
6
6
6
3
6
5
3
4
7
7
4
7
6
9
8
7
8
5
6
6
7
1
6
8
5
12
8
9
5
5
4
9
6
4
4
9
9
5
7
3
10
11
13
9
9
3
5
5
15
8
10
7
9
11
8
8
12
9
5
7
2
8
12
7
12
9
19
9
12
13
11
11
14
4
9
4
12
12
7
11
11
10
8
13
13
8
14
10
14
11
21
19
10
11
20
9
17
20
15
15
15
9
11
20
12
13
19
13
16
24
15
18
17
18
24
13
22
24
13
25
18
21
23
28
18
32
20
29
29
22
25
32
27
25
21
31
32
23
32
35
34
31
28
36
37
35
38
37
38
32
39
46
40
51
35
29
42
42
45
45
39
52
48
62
48
36
38
47
58
52
47
44
40
45
44
53
35
50
44
47
44
37
37
41
35
48
40
40
36
35
33
33
39
35
36
37
33
33
35
40
39
35
41
28
38
26
46
24
34
21
43
33
30
36
30
31
22
34
32
16
18
27
22
32
30
30
31
25
22
27
23
20
21
35
25
29
31
27
21
27
15
20
26
26
33
17
28
24
25
22
37
21
21
30
27
34
26
31
43
30
41
30
40
37
55
41
49
60
64
83
65
65
67
54
59
65
69
60
76
71
76
68
61
53
49
55
49
52
41
48
53
43
37
44
37
28
33
24
23
20
17
12
11
8
11
4
3
5
1
1
3
2
2
2
1
2
0
1
1
0
1
0
1
1
0
0
0
1
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
1
0
0
0
File: C:\xrfdaten\2014KW30\chris\std30kev\std10026.spe
Start: 22.07.2014 10:01:18
Stop: 22.07.2014 10:02:59
BESSY: 298.153382 298.9709553
IONI: 2.003979E-11 2.006964E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 98036
Run: 100000
Dead: 2
ICR: 170
OCR: 0
Accumulated BESSY: 59654.1844243
Accumulated Ioni: 4.00950098E-09
N Accumulation: 200
Normfactor:  5.0880818e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Fe
 
exp_x 152.720397949219 152.7204
exp_y 150.465789794922 150.4658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  26   1      6.399         3.         5.        .76
