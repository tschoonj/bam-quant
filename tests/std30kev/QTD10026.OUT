     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:28:45  บ   
     บ  Spectrum: QTD10026.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 -1800        2 iterations done    บ   
     บ  ChiSquare =     6.0   last change =   .00%    lambda= 1.E-04   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -53.1 ๑   100.0   บ     -53.1 ๑    97.7   บ   
     บ  GAIN (eV/ch)  บ     20.869 ๑   2.000   บ    20.869 ๑    .715   บ   
     บ det NOISE (eV) บ      222.8 ๑    40.0   บ     222.8 ๑    63.5   บ   
     บ  FANO factor   บ       .047 ๑    .050   บ      .047 ๑    .050   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Fe-Ka                                 3. ๑      5.           บ   
     บ    KA1         6.399   1.00000           3. ๑      6.       .8  บ   
     บ              309.164    236.76          16.           9.17E-01  บ   
     บ                                                                 บ   
     บ  2 Fe-Kb                                -1. ๑      4.           บ   
     บ    KB1         7.059   1.00000          -1. ๑      6.      1.0  บ   
     บ              340.789    238.15          16.           9.38E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                               0. ๑      0.           บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
